docs/public-networks/reference/api/index.md

```diff
@@ -2775,6 +2775,194 @@ curl -X POST -H "Content-Type: application/json" --data '{ "query": "{block (num
 
 <!--/tabs-->
 
+### `eth_getBlockReceipts`
+
+This JSON-RPC method returns all transaction receipts for a given block and takes the block identifier as a parameter. Transaction receipts are useful because they provide a way to track the success or failure of a transaction (`1` if successful and `0` if failed), as well as the amount of gas used and any event logs that might have been produced by a smart contract during the transaction.
+
+Transaction receipts also contain the transaction hash, the block number and hash in which the transaction was included, and the contract address of any contract created as a result of the transaction.
```

Comment:
```suggestion
Returns all transaction receipts for a given block. Transaction receipts provide a way to track the success or failure of a transaction (`1` if successful and `0` if failed), as well as the amount of gas used and any event logs that might have been produced by a smart contract during the transaction.
```

If you look at other API method descriptions on this reference page, we typically keep them short. Developers looking at this page will already have an idea of what transaction receipts are, and they can learn exactly what they contain by clicking through to the linked transaction objects reference.